dwave/preprocessing/composites/spin_reversal_transform.py

```diff
@@ -48,6 +48,9 @@ class SpinReversalTransformComposite(dimod.core.Sampler, dimod.core.Composite):
 
         seed: As passed to :func:`numpy.random.default_rng`.
 
+    .. versionadded:: 0.6.8
+        Support for context manager protocol and :meth:`.close` method.
+
```

Comment:
line 58: sampler = `composed_sampler = SpinReversalTransformComposite(base_sampler)` opened outside context manager and not closed